web-src/src/main-app/components/scripts/ScriptListItem.vue

```diff
@@ -96,26 +105,45 @@ export default {
 }
 
 .scripts-list .collection-item .menu-item-state.executing,
-.scripts-list .collection-item .menu-item-state.finished {
+.scripts-list .collection-item .menu-item-state.finished,
+.scripts-list .collection-item .menu-item-state.cannot-parse {
   display: inline;
 }
 
-.scripts-list .collection-item .menu-item-state.executing > .check-icon {
+/* state: executing */
+
+.scripts-list .collection-item .menu-item-state.executing > .check-icon,
+.scripts-list .collection-item .menu-item-state.executing > .failed-icon {
   display: none;
 }
 
 .scripts-list .collection-item .menu-item-state.executing > .preloader-wrapper {
   display: block;
 }
 
+/* state: finished */
+
 .scripts-list .collection-item .menu-item-state.finished > .check-icon {
   display: block;
 }
 
-.scripts-list .collection-item .menu-item-state.finished > .preloader-wrapper {
+.scripts-list .collection-item .menu-item-state.finished > .preloader-wrapper,
+.scripts-list .collection-item .menu-item-state.finished > .failed-icon {
```

Comment:
I think there is no need to duplicate this `display:none` multiple times. 
You can define it once, smth like:
```
.scripts-list .collection-item .menu-item-state > .failed-icon {
  display: none;
}
```

And it should be enough.
I do not know why I copied it multiple times for check icon and preload wrapper.

Reply:
Not quite sure what do you mean, sorry.